### ISSUE TITLE
Fresh install fails

### ISSUE THREAD

**Issue by gridworkz** [fairnesscoop/permacoop#131]
I just cloned a fresh install and I get the following when I do `make install`

osboxes@osboxes:~/permacoop$ **sudo make install**
cp server/ormconfig.json.dist server/ormconfig.json
cp server/.env.dist server/.env
cp client/config.js.dist client/config.js
docker run -it --rm -v /server:/app -w /app node npm i
npm WARN saveError ENOENT: no such file or directory, open '/app/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/app/package.json'
npm WARN app No description
npm WARN app No repository field.
npm WARN app No README data
npm WARN app No license field.

up to date in 0.421s
found 0 vulnerabilities

docker run -it --rm -v /client:/app -w /app node npm i
npm WARN saveError ENOENT: no such file or directory, open '/app/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/app/package.json'
npm WARN app No description
npm WARN app No repository field.
npm WARN app No README data
npm WARN app No license field.

up to date in 0.391s
found 0 vulnerabilities

make start
make[1]: Entering directory '/home/osboxes/permacoop'
docker-compose -p permacoop up -d
Creating permacoop_database_1 ... done
Creating permacoop_api_1      ... done
Creating permacoop_client_1   ... done
make[1]: Leaving directory '/home/osboxes/permacoop'
make api-build-dist
make[1]: Entering directory '/home/osboxes/permacoop'
docker-compose -p permacoop exec api npm run build
ERROR: No container found for api_1
Makefile:36: recipe for target 'api-build-dist' failed
make[1]: *** [api-build-dist] Error 1
make[1]: Leaving directory '/home/osboxes/permacoop'
Makefile:10: recipe for target 'install' failed
make: *** [install] Error 2

`docker ps` shows the database ok but the client and the api containers exit immediately. - Cheers, Dave

**Comment by mmarchois** [comment]
Hi,

I'm going to take a look at this, thank you for the feedback.

**Comment by mmarchois** [comment]
Hi @gridworkz,

Can you try again?

```bash
make stop
make rm
docker volume prune (ro remove database volume)
make install
``` 
I have added a proxy, so you can access to the api with the url `http://localhost/api` and the client  with `http://localhost`.

Let me know if you are still having problems.

**Comment by gridworkz** [comment]
Hi Mathieu:

I got the client/server started: Couple things to note -

1) change host to localhost in ./server/ormconfig.json (connect to Pg)
2) change host to localhost in ./client/config.js (connect to Redis)

So after a fresh install I still don't see john@foe.com in the database. I
can see the queries being executed and migrated OK during the server
startup. If I exec into the db docker - 'select * from user' just returns
docker and 'select * from user_administravie' is empty. I tried to login
with john@doe.com / john and still no luck. I'm still a little uncertain
where the api token comes in during login.

Cheers,
Dave
